docs/conceptual/option.filter['t]-function-[fsharp].md

```diff
@@ -0,0 +1,60 @@
+---
+title: Option.filter<'T> Function (F#)
+description: Option.filter<'T> Function (F#)
+keywords: visual f#, f#, functional programming
+author: eriawan
+manager: danielfe
+ms.date: 07/01/2016
+ms.topic: language-reference
+ms.prod: visual-studio-dev14
+ms.technology: devlang-fsharp
+ms.assetid: ff36174c-0820-47e4-b7ce-6fa93f19bcb5 
+---
+
+# Option.filter<'T> Function (F#)
+
+Invokes a function on an optional value that itself yields an option.
+
+**Namespace/Module Path**: Microsoft.FSharp.Core.Option
+
+**Assembly**: FSharp.Core (in FSharp.Core.dll)
+
+
+## Syntax
+
+```
```

Comment:
Thanks @dend 

I'm now also adding `fsharp` syntax identifier for all functions in `option`, not just `option.filter` because most of them doesn't have `fsharp` syntax identififer.

Currently I'm still adding doc for new APIs including Option.toNullable. I'll add `fsharp` syntax identifier as well to the rest of the new APIs doc.